pkg/mcs/registry/registry.go

```diff
@@ -22,7 +21,7 @@ import (
 	"net/http"
 
 	"github.com/pingcap/log"
-	"github.com/tikv/pd/server"
+	bs "github.com/tikv/pd/pkg/basic_server"
```

Comment:
```suggestion
	bs "github.com/tikv/pd/pkg/basicserver"
```

Reply:
how about basicsvr?

Reply:
I also think `basicserver` is better than `basicsvr`